modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Iroha2Client.kt

```diff
@@ -44,6 +45,7 @@ import jp.co.soramitsu.iroha2.generated.datamodel.events.pipeline.EventFilter as
 
 open class Iroha2Client(
     open var peerUrl: URL,
+    open var telemetryUrl: URL? = null,
```

Comment:
`telemetryUrl` is nullable, if it is not set, then calling `status` will have strange behaviour. As far as I understand telemetry will be on the other port, but always on the same host with `peerUrl`. I think we can create `telemetryUrl` from `peerUrl`
```suggestion
open var telemetryUrl: URL = URL(peerUrl.protocol, peerUrl.host, DEFAULT_TELEMETRY_PORT, peerUrl.file),
```